tests/integration/test_opensearch.py

```diff
@@ -82,9 +82,9 @@ async def test_deploy(ops_test: OpsTest, app_charm: PosixPath, data_integrator_c
             OPENSEARCH[ops_test.cloud_name],
             channel="edge",
             application_name=OPENSEARCH[ops_test.cloud_name],
-            num_units=3,
+            num_units=2,
```

Comment:
We keep getting a sharding error with 3 units, I'm discussing with Mehdi if this is to be addressed as an issue (separate Jira ticket) on Opensearch side.

Reply:
The conclusion is that we should follow this up on Opensearch side.

https://warthogs.atlassian.net/browse/DPE-3445